resources/auth.py

```diff
@@ -13,7 +13,36 @@ def is_general_admin(user):
 
 
 def is_any_admin(user):
-    return is_authenticated_user(user) and (
-        is_general_admin(user) or
-        user.unit_group_authorizations.admin_level().exists() or
-        user.unit_authorizations.admin_level().exists())
+    from .enums import UnitGroupAuthorizationLevel, UnitAuthorizationLevel
```

Comment:
Is there a specific reason for the imports at function level?

Reply:
Not really. I changed them to be at module level.